RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -535,7 +535,8 @@ private IEnumerable<IMenuItem> GetFormDesignerContextMenuItems()
         {
             return new IMenuItem[]
             {
-                KernelInstance.Get<FormDesignerRefactorRenameCommandMenuItem>()
+                KernelInstance.Get<FormDesignerRefactorRenameCommandMenuItem>(),
+                KernelInstance.Get<FormDesignerFindAllReferencesCommandMenuItem>()
```

Comment:
@MDoerner will these be a problem? or note has to be taken of them I guess?

Reply:
I just have to make a note that it is there so I do not forget that this is the confict I am having for `RubberduckModule`.